### PR TITLE
solaar: 2018-02-02 -> 2019-01-30 + udev rules inclusion

### PR DESCRIPTION
**Pull request by samueldr**
###### Motivation for this change

Includes the udev rules in the solaar package. This allows the following to work.

```nix
{ pkgs, ... }:
{
  services.udev.packages = [ pkgs.solaar ];
}
```

In addition, update to the latest commit.

###### Things done

- ☑ Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - ☑ NixOS
- ☑ Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- ☑ Tested execution of all binary files (usually in `./result/bin/`)
- ☑ Assured whether relevant documentation is up to date
- ☑ Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/solaar/default.nix [hunk 1]
```diff
@@ -18,6 +18,11 @@ pythonPackages.buildPythonApplication rec {
     wrapProgram "$out/bin/solaar-cli" \
       --prefix PYTHONPATH : "$PYTHONPATH" \
       --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH"
+
+    mkdir -p $out/lib/udev/rules.d
+    for f in rules.d/*.rules; do
+      cp "$f" $out/lib/udev/rules.d/
```
Comment (cleverca22) [inline pkgs/applications/misc/solaar/default.nix]: what about `cp -r rules.d $out/lib/udev` ?

Comment (samueldr) [reply pkgs/applications/misc/solaar/default.nix]: there's an install.sh in the directory :)

Comment (cleverca22) [reply pkgs/applications/misc/solaar/default.nix]: ah, then it would be `cp rules.d/*.rules $out/lib/udev/rules.d/`

**Comment by samueldr** [comment]
Also took the opportunity to cleanup using pname/version and removing a now unused attribute.

From nixpkgs manual:

>  namePrefix: Prepends text to ${name} parameter. In case of libraries, this defaults to "python3.5-" for Python 3.5, etc., and in case of applications to "".

**Pull request merged**